### ISSUE TITLE
Choice fields with hyphens result in dashes in enum values

### ISSUE THREAD

**Issue by PeterJCLaw** [rudderlabs/rudder-typer#92]
For example:

```json
"shipping_method": {
	"enum": [
		"fedex",
		"dhl",
		"royal-mail",
		"carrier-pigeon"
	],
	"type": [
		"string"
	]
},
```

Results in:
```
/**
 * This client was automatically generated by RudderTyper. ** Do Not Edit **
 */
package com.rudderstack.ruddertyper.generated;


public enum ShippingMethod {

    S_FEDEX("fedex"),
    S_DHL("dhl"),
    S_ROYAL-MAIL("royal-mail"),
    S_CARRIER-PIGEON("carrier-pigeon");

    private final Object value;

    ShippingMethod(Object value) {
        this.value = value;
    }

    public Object getValue() {
        return value;
    }
}
```
This appears to affect at least the Java and TypeScript generators.

**Comment by TWiStErRob** [comment]
This line is preparing the name (Java):

https://github.com/rudderlabs/rudder-typer/blob/921f5e4ae8c9f909d7bc738193b4be8ef961ac37/src/generators/android/android.ts#L167

it only removes "space" as invalid characters, it should probably remove everything that's valid according to

https://github.com/rudderlabs/rudder-typer/blob/921f5e4ae8c9f909d7bc738193b4be8ef961ac37/src/generators/android/android.ts#L60-L61

**Comment by PeterJCLaw** [comment]
IMO it would be nice to replace any (string of) invalid characters with `_` rather than dropping them entirely -- so we get like `SOME_LONG_NAME` from `some-long-name` rather than `SOMELONGNAME`, as the former is more readable. Appreciate that changing this for spaces might be breaking for existing consumers though.

**Comment by TWiStErRob** [comment]
Oh, yeah, definitely! Removal is hard to read, sorry.

**Comment by akashrpo** [comment]
This issue has been resolved in #94. Closing this now.
